src/util/Angle.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Util class for angle related calculations.
+ */
+const AngleUtil = {
+  /**
+   * Determine the angle between two coordinates. The angle will be between
+   * -180° and 180°, with 0° being in the east. The angle will increase
+   * counter-clockwise.
+   *
+   * Inspired by http://stackoverflow.com/a/31136507
+   *
+   * @param {Array<Number>} start The start coordinates of the line with the
+   *   x-coordinate being at index `0` and y-coordinate being at index `1`.
+   * @param {Array<Number>} end The end coordinates of the line with the
```

Comment:
maybe here as type an OpenLayers coordinate can be used: https://openlayers.org/en/latest/apidoc/module-ol_coordinate.html#~Coordinate